elizabeth/utils.py

```diff
@@ -146,3 +147,21 @@ def update_dict(initial, other):
         else:
             initial[key] = other[key]
     return initial
+
+def check_gender(gender):
+    """
+    Check a given gender if it's valid
+
+    Valid genders are 'male', 'female' and None. If None is given, a gender
+    is randomly choosen.
+
+    :param gender: str or None
+    :return: One of 'male' or 'female'
+    :rtype: str
+    """
+    if gender is None:
+        return choice(['male', 'female'])
+    gender = gender.lower()
+    if gender not in ['male', 'female', None]:
```

Comment:
None values are caught by the first if condition, so gender will never be equal to None here